Dockerfile

```diff
@@ -19,7 +19,7 @@ FROM --platform=$TARGETPLATFORM bitnami/kubectl:latest AS final
 
 WORKDIR /src
 
-ENV SECRET_NAME=bux-keys
+ENV SECRET_NAME=bsv-keys
```

Comment:
Shouldn't these be `spv-keys` ?

Reply:
rather `spv-wallet-keys`

Reply:
`spv-wallet-keys` I updated it with this name.